query-engine/core/src/schema/enum_type.rs

```diff
@@ -99,7 +99,13 @@ impl DatabaseEnumType {
         self.internal_enum
             .values
             .iter()
-            .map(|v| v.name.to_string())
+            .map(|v| {
```

Comment:
`v.database_name.as_ref().unwrap_or(&v.name).clone()` or similar instead of returns.

Reply:
Damn it, I knew there was such a clean way to do it. Ty!